recipes/chezmoi

```diff
@@ -1,2 +1,3 @@
 (chezmoi :fetcher github
-         :repo "tuh8888/chezmoi.el")
+         :repo "tuh8888/chezmoi.el"
+         :files ("*.el" "extensions"))
```

Comment:
Don't use `*.el` if you can help it (this will include things like dir-locals.el).  If this works for you I'd recommend:

```elisp
(chezmoi :fetcher github
         :repo "tuh8888/chezmoi.el"
         :files (:defaults "extensions"))
```